components/actions/ActionsTable.js

```diff
@@ -54,13 +54,9 @@ const ACTION_ROW_FRAGMENT = gql`
 const Status = (props) => {
   const { action, plan } = props;
   const checkedStatus = cleanActionStatus(action, plan.actionStatuses);
-
-  return (
-    <StatusBadge
-      plan={plan}
-      statusSummary={getStatusSummary(plan, action.statusSummary)}
-    />
-  );
+  const statusSummary = getStatusSummary(plan, action.statusSummary);
+  const actionWithStatusSummary = Object.assign({}, action, { statusSummary });
```

Comment:
Nitpick: They're very similar in practice, but I tend to prefer spreading objects over `Object.assign`. Mainly because it's easy to accidentally mutate an object by forgetting the empty object as the first argument, whereas object spreading always creates a new one. I think it's a bit more readable too, but that's personal preference

```suggestion
  const actionWithStatusSummary = { ...action, statusSummary };
```

Reply:
We have been using Object.assign elsewhere, I'll leave this now but let's reach a conclusion later.